contextualhelp/data.json

```diff
@@ -209,6 +213,10 @@
                   "title": "Dove posso salvare il certificato sull'app IO?",
                   "body": "Premendo il pulsante 'Salva', puoi salvare il QR code del certificato nella galleria immagini del tuo dispositivo per poterlo usare anche se non sei connesso ad internet."
                },
+               {
+                  "title": "Posso ricevere su IO la Certificazione di esenzione dalla vaccinazione anti-COVID-19?",
+                  "body": "Sì, puoi ricevere e visualizzare il tuo Certificato di esenzione direttamente in app.\n Questa certificazione è valida solo in Italia fino alla data indicata sulla certificazione e può essere utilizzata per accedere dove è richiesta una Certificazione Verde COVID-19.\n Per maggiori informazioni sulla Certificazione di esenzione dalla vaccinazione anti-COVID-19, consulta le FAQ dedicate [https://www.dgc.gov.it/web/esenzione.html](https://www.dgc.gov.it/web/esenzione.html)."
```

Comment:
I'm getting an error with this URL, is it correct @mariateresaventura ?
![Uploading Screenshot 2022-02-15 at 17.44.13.png…]()

Reply:
I check it, the correct url is as follows: https://www.dgc.gov.it/web/esenzione.html